### PR TITLE
Update database-migrations.html.md.erb

### PR DESCRIPTION
**Pull request by olimart**
@paulcsmith I let you reword it if necessary.

The downside is every time a new extension is added, one must remember to update this page. Maybe it's not the ideal solution but being able to know your options would be nice.
(I tried text for instance).

### CODE REVIEW

File: source/guides/database-migrations.html.md.erb [hunk 1]
```diff
@@ -34,6 +34,8 @@ create :users do
 end
 ```
 
+Currently supporting `String, Int32, Bool, and Time`.
```
Comment (paulcsmith) [inline source/guides/database-migrations.html.md.erb]: Can you add a `> ` to the beginning of this and add `Float` to this list? Also I think `"Lucky currently supports..."` might read a bit better.

**Review by paulcsmith — COMMENTED** [review]
Thanks for this. That will definitely help clarify things for people 🎉

**Comment by olimart** [comment]
@paulcsmith I have updated the PR accordingly.

**Comment by paulcsmith** [comment]
Awesome! Thanks for adding this clarification 👍

**Pull request merged**